pkg/operator/ceph/object/config.go

```diff
@@ -44,6 +44,10 @@ caps osd = "allow rwx"
 	caBundleExtractedDir            = caBundleTrustedDir + "extracted/"
 	caBundleKeyName                 = "cabundle"
 	caBundleFileName                = "custom-ca-bundle.crt"
+	ldapVolumeName                  = "rook-ceph-rgw-ldap"
+	ldapDir                         = "/etc/ceph/ldap"
```

Comment:
How about a different location for these files? Documents I found suggest `/etc/` is often used, which we also shouldn't use. I think `/etc/ldap` would be betters.